app/views/account_reset/pending/confirm.html.erb

```diff
@@ -1,11 +1,17 @@
 <% self.title = t('account_reset.cancel_request.title') %>
 
+<%= render PageHeadingComponent.new.with_content(t('account_reset.pending.header')) %>
+
 <p><%= t('account_reset.pending.confirm', interval: @account_reset_deletion_period_interval) %></p>
 
-<%= button_to(
-      account_reset_pending_cancel_path,
-      class: 'usa-button usa-button--wide usa-button--big margin-bottom-2',
+<%= render ButtonComponent.new(
+      url: account_reset_pending_cancel_path,
       method: :post,
-    ) { t('forms.buttons.continue') } %>
+      class: 'margin-top-3 margin-bottom-2',
+      wide: true,
+      big: true,
+    ).with_content(t('account_reset.pending.cancel_request')) %>
 
-<%= link_to(t('links.go_back'), account_reset_pending_path) %>
+  <div>
+    <%= link_to(t('links.go_back'), account_reset_pending_path) %>
+  </div>
```

Comment:
Nit: Can we fix up the indentation and include a final newline?

```suggestion
<div>
  <%= link_to(t('links.go_back'), account_reset_pending_path) %>
</div>

```

Reply:
I wonder why the lint didn't catch the missing newline because it usually catches that 🤔

Reply:
I think [ERB Lint](https://github.com/Shopify/erb_lint) mainly lints Ruby code inside `<%` and `%>`, not HTML markup itself. Though yea, would be nice if we could find a way to enforce consistency with this. 👍